recipes/soplex/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 3.8)
+project(test_package CXX)
+set(CMAKE_CXX_STANDARD 14)
+
+find_package(soplex REQUIRED CONFIG)
+add_executable(${PROJECT_NAME} main.cpp)
+target_link_libraries(${PROJECT_NAME} PRIVATE soplex::soplex)
```

Comment:
I see the project install targets without a namespace https://github.com/scipopt/soplex/blob/4ab99fc9f52fcbcc738a90e59cbba1862eb71d92/src/CMakeLists.txt#L223

We should add to the package info to do that same?

Reply:
I do not understand this comment. Is it about changing `soplex::soplex` to `soplex`? Is that required?

Reply:
Yes, we try to make ConanCenter seamless as possible, so we aim to match upstream

Reply:
I changed the target name

Reply:
@prince-chrismc I changed the target name, but now test package v1 does not compile anymore. I need help with that